bot/gif_command.go

```diff
@@ -45,13 +45,13 @@ func NewGIFCommand(endpoint, key string, w io.Writer) cli.Command {
 			query := url.Values{}
 			query.Set("key", key)
 			query.Set("q", strings.Join(args, " "))
-			query.Set("limit", "1")
```

Comment:
Limit should also be an integer flag with the default value shown in their [docs](https://tenor.com/gifapi/documentation#endpoints-search): 20